### PR TITLE
Closes #1 create index.html murkup

### LINKED ISSUE #1: create index.html murkup

### PR DESCRIPTION
**Pull request by dgzargo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (AHrysyuk) [inline index.html]: your code is shit

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (TarasZadilskyi) [inline index.html]: pl.change

File: index.html (hunk 1 above)
Comment (P4NCH3R) [reply index.html]: ++++

Comment (RishkoPavel) [reply index.html]: lya

Comment (P4NCH3R) [reply index.html]: $to-takoe ?

**Comment by PanasiukAndriy** [comment]
nice

File: index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+
```
Comment (AlexBoichuk) [inline index.html]: No content

File: index.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (PanasiukAndriy) [inline index.html]: change

**Pull request merged**